backend/pubsub.js

```diff
@@ -395,11 +424,19 @@ const publicMethods = {
   },
 
   // Enumerates the subscribers of a given channel.
-  * enumerateSubscribers (channelID: string): Iterable<Object> {
+  * enumerateSubscribers (channelID: string, kvKey?: string): Iterable<Object> {
     const server = this
 
     if (channelID in server.subscribersByChannelID) {
-      yield * server.subscribersByChannelID[channelID]
+      const subscribers = server.subscribersByChannelID[channelID]
+      if (!kvKey) {
+        yield * subscribers
+      } else {
+        for (const subscriber of subscribers) {
+          const kvFilter = subscriber.kvFilter.get(kvKey)
+          if (!kvFilter || kvFilter.has(kvKey)) yield subscriber
+        }
```

Comment:
This should be `channelID` instead of `kvKey`, right?

```js
for (const subscriber of subscribers) {
  const filterSet = subscriber.kvFilter.get(channelID)
  if (!filterSet || filterSet.has(kvKey)) yield subscriber
}
```

Reply:
You're right. Fixed.